pkg/vsphere/spec/spec_test.go

```diff
@@ -29,7 +29,7 @@ import (
 func URL(t *testing.T) string {
```

Comment:
While you're in there, looks like this func should be removed and the test changed to use 'test.URL()' - via import of pkg/vsphere/test.